### ISSUE TITLE
allow hiding pois/polys by type

### ISSUE THREAD

**Issue by namdre** [eclipse/sumo#11305]
maybe with an sub-dialog of the gui-settings dialog that generates checkboxes based on all know poi/poly types

**Comment by palvarezlopez** [comment]
Also in NETEDIT?

**Comment by namdre** [comment]
Yes. Though I don't think this will need extra code in netedit. It should be fully contained within guisettings and GUIPolygon / GUIPointOfInterest.
